app/views/FIAlreadyRemovedView.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components._
+
+@this(
+        layout: templates.Layout,
+        govukButton: GovukButton,
+        link: Link,
+        heading: Heading
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("fIAlreadyRemoved.title")), showBackLink = false) {
+
+    @heading(messages("fIAlreadyRemoved.heading"))
+
+    <p class="govuk-body govuk-!-margin-bottom-2">@link(controllers.routes.YourFinancialInstitutionsController.onPageLoad().url, "fIAlreadyRemoved.p1")</p>
```

Comment:
i think you can add classes to the Link component without needing to wrap it in a <p>